cypress/integration/overlays/complex-bottom-sheet.spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
+
+const { should } = require("chai");
+
+
+describe('Functional complex bottom sheet', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/complex-bottom-sheet');
+    });
+
+    it('title displays', () => {
+        cy.get('[data-cy=pxb-toolbar]').should('contain', 'Complex Bottom Sheet');
+    });
+
+    it('bottom sheet opens & action items display', () => {
+        cy.get('[data-cy=action-menu]').click()
+        cy.get('[data-cy=btm-sheet-sort]').should('contain', 'Sort By')
+        .and('contain', 'Time')
+        .and('contain', 'Type')
+        cy.get('[data-cy=btm-sheet-show]').should('contain', 'Show')
+        .and('contain', 'Active Alarms')
+        .and('contain', 'alarms')
+        .and('contain', 'settings')
+        .and('contain', 'session')
+        cy.get('[data-cy=btm-sheet-cancel]').should('contain', 'Close')
+    });

```

Comment:
Should we add a test to verify that the sorting and filtering works?

Reply:
Unit tests are covering this really good but I can add chai-sorted package for cypress to grab all items in array and verify if needed

Reply:
No, good point. If the unit testing is covering this well, no need to double dip.